extensions/wikia/PortableInfobox/services/PortableInfoboxDataService.class.php

```diff
@@ -63,4 +58,68 @@ public function getImages() {
 
 		return array_keys( $images );
 	}
+
+	/**
+	 * @desc For given Title, get property 'infoboxes' from parser output. If property is empty, this may mean that
+	 * template is inside the <noinclude> tag. In this case, we want to skip the <includeonly> tags, get from this only
+	 * infoboxes and parse them again to check their presence and get params.
+	 * @param $title Title
+	 * @return mixed
+	 */
+	protected function getParsedInfoboxes( $title ) {
+		$article = Article::newFromTitle( $title, RequestContext::getMain() );
+
+		if ( $title->getNamespace() === NS_TEMPLATE ) {
+			$templateText = $article->fetchContent();
+			$includeonlyText = $this->getIncludeonlyText( $templateText );
+
+			if ( $includeonlyText ) {
+				$parser = new Parser();
+				$parserOptions = new ParserOptions();
+				$frame = $parser->getPreprocessor()->newFrame();
+
+				$templateTextWithoutIncludeonly = $parser->getPreloadText( $includeonlyText, $article->getTitle(), $parserOptions );
+				$infoboxes = $this->getInfoboxes( $templateTextWithoutIncludeonly );
+
+				if ( $infoboxes ) {
+					foreach ( $infoboxes as $infobox ) {
+						PortableInfoboxParserTagController::getInstance()->render( $infobox, $parser, $frame );
+					}
+					return $parser->getOutput()->getProperty( self::INFOBOXES_PROPERTY_NAME );
+				}
+			}
+		}
+
+		//on empty parser cache this should be regenerated, see WikiPage.php:2996
+		$parserOutput = $article->getParserOutput();
+		$parsedInfoboxes = $parserOutput ?
+			$parserOutput->getProperty( self::INFOBOXES_PROPERTY_NAME )
+			: false;
+
+		return $parsedInfoboxes;
+	}
+
+	/**
+	 * @desc From the template without <includeonly> tags, creates an array of
+	 * strings containing only infoboxes. All template content which is not an infobox is removed.
+	 *
+	 * @param $text string Content of template which uses the <includeonly> tags
+	 * @return array of striped infoboxes ready to parse
+	 */
+	protected function getInfoboxes( $text ) {
+		preg_match_all( "/<infobox.+<\/infobox>/sU", $text, $result );
```

Comment:
It will appear only in case when inside the `<includeonly>` tags will contain the `<nowiki>` tag which will include the `<infobox>` `</infobox>` tags and between them strange, invalid wikitext which actually is not a real infobox.

While checking how will it really behave in this case, I found an interesting issue in out Infobox Parser which I've created the ticket for: https://wikia-inc.atlassian.net/browse/DAT-3141
so basically for now, templates like this will not parse and shouldn't appear. I'll however add a comment to check this out when the DAT-3141 will be fixed and cases like this will be possible. Thanks!